docs/installation/autoware/docker-installation.md

```diff
@@ -47,15 +49,23 @@ You might need to log out and log back to make the current user able to use dock
 
 1. Launch a Docker container.
 
-   ```bash
-   rocker --nvidia --x11 --user --volume $HOME/autoware -- ghcr.io/autowarefoundation/autoware-universe:latest
-   ```
+   - For amd64 architecture computers with NVIDIA GPU:
 
-   If you want to run container without using NVIDIA GPU:
+     ```bash
+     rocker --nvidia --x11 --user --volume $HOME/autoware -- ghcr.io/autowarefoundation/autoware-universe:latest
+     ```
 
-   ```bash
-   rocker -e LIBGL_ALWAYS_SOFTWARE=1 --x11 --user --volume $HOME/autoware -- ghcr.io/autowarefoundation/autoware-universe:latest
-   ```
+   - If you want to run container without using NVIDIA GPU:
+
+     ```bash
+     rocker -e LIBGL_ALWAYS_SOFTWARE=1 --x11 --user --volume $HOME/autoware -- ghcr.io/autowarefoundation/autoware-universe:latest
+     ```
+
+   - For arm64 architecture computers:
+
+     ```bash
+     rocker -e LIBGL_ALWAYS_SOFTWARE=1 --x11 --user --volume $HOME/autoware -- ghcr.io/autowarefoundation/autoware-universe:latest-arm64
```

Comment:
No-suffix images (e.g. `latest`) are multi architecture. Why do you need this? :thinking: 
https://github.com/autowarefoundation/autoware/pkgs/container/autoware-universe/21020592?tag=latest
![image](https://user-images.githubusercontent.com/31987104/169770385-26bb2694-e2b3-49bb-a129-5045ec810fef.png)

Reply:
Yes you're right, it was based on old versions, I have already checked the latest and its working. I will update.

Reply:
I see. Sometimes if CI fails, the combined images may not contain both architectures.